### ISSUE TITLE
LICENSE

### ISSUE THREAD

**Issue by tofumatt** [mozilla/node-firefox#10]
Is this code supposed to be MPL or Apache-licensed? A lot of the modules are Apache and that seems to be a better fit for node (also it's way more straightforward, copyleft is confusing).

Should we change the node-firefox license to Apache 2.0?

**Comment by sole** [comment]
Yeah I mean this module doesn't have much (none?) actual code, but yes, let's have the same license in all!
